### DISCUSSION TITLE
It’s possible to record 24/7?

### DISCUSSION THREAD

**Discussion by orebla85** [roleoroleo/yi-hack-MStar#428] [General]
Hi,
This is a best hack to use a low cost ipcam to use it to the top.
But it’s possible to record video 24/7 like cloude service by Yi?
I didn't see any options, just the ability to record a maximum of 60 seconds.
It’s right?
Thanks

**Comment by roleoroleo** [comment]
I didn't test it but if you set
"Disable Cloud" - ON
"Recording without Cloud" - ON
"Save video when a motion is detected" - OFF
it should work.

**Comment by orebla85** [comment]
Thanks for the answer. I'll try and I'll tell you!
Because it would be a really great function to record 24/7 and save to Nas!
